modules/setting/setting.go

```diff
@@ -705,18 +718,13 @@ func NewContext() {
 	if err = sec.MapTo(&LFS); err != nil {
 		log.Fatal("Failed to map LFS settings: %v", err)
 	}
-	LFS.ContentPath = sec.Key("LFS_CONTENT_PATH").MustString(filepath.Join(AppDataPath, "lfs"))
-	if !filepath.IsAbs(LFS.ContentPath) {
-		LFS.ContentPath = filepath.Join(AppWorkPath, LFS.ContentPath)
-	}
+	LFS.ContentPath = sec.Key("LFS_CONTENT_PATH").MustString(filepath.Join(appDataUserPath, "lfs"))
```

Comment:
Why this changed?

Reply:
Because `storage.OpenBucket` handles both absolute and relative path for us.